projects/experimental/directives/pin/pin.directive.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Directive,
+    EventEmitter,
+    HostListener,
+    Inject,
+    Input,
+    Output,
+    ViewEncapsulation,
+} from '@angular/core';
+import {TUI_PLATFORM, TuiDirectiveStylesService, TuiPlatform} from '@taiga-ui/cdk';
+
+@Component({
+    host: {class: 'tui-pin'},
+    template: '',
+    styleUrls: ['./pin.styles.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    encapsulation: ViewEncapsulation.None,
+})
+export class TuiPinStyleComponent {}
+
+@Directive({
+    selector: '[tuiPin]',
+    host: {
+        tuiPin: '',
+        '[attr.tabindex]': '0',
+        '[class._open]': 'open',
+        '[attr.data-platform]': 'platform',
+    },
+})
+export class TuiPinDirective {
+    @Input('tuiPin')
+    open: boolean | '' = false;
+
+    @Output('tuiPinChange')
+    openChange = new EventEmitter<boolean>();
+
+    @Output()
+    focusChange = new EventEmitter<boolean>();
```

Comment:
If we drop pin group directive from first iteration, let's drop focus related stuff too. It's not needed without it and we can always add it later.

Reply:
fixed